testing/common_testing_util.py

```diff
@@ -76,4 +84,17 @@ def remove_dataset(dataset: str = 'data_testing'):
 
 def remove_tmp_dir():
     """Recursively remove the ``tmp`` directory if it exists."""
-    shutil.rmtree(path_temp, ignore_errors=True)
+    count = 0
+    for element in list(path_temp.iterdir()):
+        if element.is_dir() and element.name.startswith("tmp"):
+            count += 1
+
+    if count > 100:
+        num_to_delete = count - 100
+        lst = sorted(path_temp.iterdir(), key=os.path.getctime)
+        i = 0
+        while num_to_delete:
+            if lst[i].is_dir() and lst[i].stem.startswith("tmp"):
+                shutil.rmtree(lst[i], ignore_errors=True)
+                num_to_delete -= 1
+            i += 1
```

Comment:
Note that now you are nuking entire /tmp whenever ivadomed runs a unit test.... side effect is potentially tremendous for all other linux operations that leverage /tmp.  This is a potential undesirable side effect.